rekor-rs/src/models/log_entry.rs

```diff
@@ -0,0 +1,69 @@
+// Design a struct that stores the response returned by Rekor after making a new entry
+
+/// Stores the response returned by Rekor after making a new entry
+// get rid of default
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LogEntry {
+    // native uuid object maybe?
+    uuid: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    attestation: Option<Attestation>,
+    body: String,
+    integrated_time: i64,
+    log_i_d: String,
```

Comment:
The response we get from rekor looks like this 	
`"logID": "c0d23d6ad406973f9559f3ba2d1ca01f84147d8ffc5b8445c224f98b9591801d",`
Since rust uses snake case I named the field ad `log_i_d`.
We serialize the fields as camelCase before we use serde_json to read the response into the struct.

Reply:
Ah, gotcha. In that case, what you have is better.